app/src/main/java/com/portfolio/tasky/BaseActivity.kt

```diff
@@ -0,0 +1,50 @@
+package com.portfolio.tasky
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+
+open class BaseActivity : AppCompatActivity() {
```

Comment:
I don't recommend having Base, unless it's something ultra specific

What's your intention for using BaseActivity and BaseFragment?

Reply:
Sure, thanks for the videos!

BaseActivity and BaseFragment will accumulate all the logic to control generic components like Toolbars, logging for app traversals, independent dialogs etc.

Reply:
I recommend not doing that, instead have Util classes or helpers that handle that logic
Charfaoui makes a good video about it and how you can do it: https://www.youtube.com/watch?v=YtbwxEL4uy8

Reply:
Okay, Thanks!

Reply:
Done! Removed BaseActivity and BaseFragment from the project.